packages/paste-website/src/pages/components/progress-steps/index.mdx

```diff
@@ -0,0 +1,402 @@
+
+Only button steps may be disabled.
+
+<LivePreview
+  scope={{
+    ProgressSteps,
+    ProgressStepIncomplete,
+    ProgressStepComplete,
+    ProgressStepCurrent,
+    ProgressStepError,
+    ProgressStepSeparator,
+  }}
+  noInline
+>
+  {buttonExample}
+</LivePreview>
+
+### Progress Steps as anchors
+
+Use Progress Steps as anchors when each step links to other pages and has a unique URL.
+
+<LivePreview
+  scope={{
+    ProgressSteps,
+    ProgressStepIncomplete,
+    ProgressStepComplete,
+    ProgressStepCurrent,
+    ProgressStepError,
+    ProgressStepSeparator,
+  }}
+  noInline
+>
+  {anchorExample}
+</LivePreview>
+
+### Non-interactive Progress Steps
+
+Use Progress Steps as divs when steps have no need for user interaction.
+
+<LivePreview
+  scope={{
+    ProgressSteps,
+    ProgressStepIncomplete,
+    ProgressStepComplete,
+    ProgressStepCurrent,
+    ProgressStepError,
+    ProgressStepSeparator,
+  }}
+  noInline
+>
+  {divExample}
+</LivePreview>
+
+### Progress Steps represented vertically
+
+Use vertical Progress Steps when you’re constrained on vertical space in your layout.
+
+<LivePreview
+  scope={{
+    ProgressSteps,
+    ProgressStepIncomplete,
+    ProgressStepComplete,
+    ProgressStepCurrent,
+    ProgressStepError,
+    ProgressStepSeparator,
+  }}
+  noInline
+>
+  {verticalExample}
+</LivePreview>
+
+### Internationalization
+
+Each step within the Progress Step component has a label which can be set to adjust the icon's title for internationalization.
+
+<LivePreview
+  scope={{
+    ProgressSteps,
+    ProgressStepIncomplete,
+    ProgressStepComplete,
+    ProgressStepCurrent,
+    ProgressStepError,
+    ProgressStepSeparator,
+  }}
+  noInline
+>
+  {i18nExample}
+</LivePreview>
+
+---
+
+## Composition Notes
+
+- Each step must have a label. Keep the label concise with fewer than 5 words per label.
+- Skip articles like “a” or “the” to shorten the step.
+- Generally, start each step with a verb to keep the user grounded in the task at hand.
+- If the process has 5 steps and describing each one with a verb + noun is too long, consider shortening the step labels to just a verb, and using the in-page step label to clarify the full scope of the task. For example, use “Add” as a step label, but include “Add entry step” as the page label.
+- If the process does not fit into a mental model of performing tasks, make sure each step is phrased consistently. For example, “Business profile” and “Brand registration.”
+- Keep the placement of Progress Steps consistent between different pages (steps).
+
+### When to use an error step
+
+Show the error status on a step when it’s not the current step they’re on. When a
+user navigates to the step that has errors, use `ProgressStepCurrent` to set the status of the step to current,
+and use in-page validation to then communicate the errors in the current page.
+Check out the [Error state pattern](https://paste.twilio.design/patterns/error-state) for more guidance.
```

Comment:
```suggestion
Check out the [Error state pattern](/patterns/error-state) for more guidance.
```

ugh sorry didn't catch this in the first review. This is showing up as an external link.